### PR TITLE
Render tidy: region/AddCardReviewModal

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [[https://www.notion.so/regions-AddCardReviewModal-index-js-5c6d711153c844ecbd5f4ada124f4203](https://www.notion.so/regions-AddCardReviewModal-index-js-5c6d711153c844ecbd5f4ada124f4203)]

## Description:

No rerenders were detected. I just updated the component and it's files to match the current styleguide

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/components/regions/AddCardReviewModal/StatusHelp.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        <ul>
+          <li>
+            <Typography>
+              The project matches the specification - it does what it is
+              supposed to
+            </Typography>
+          </li>
+          <li>
+            <Typography>
+              All the relevent code is in the master branch - The master branch
```
Comment (FaithMo) [inline frontend/src/components/regions/AddCardReviewModal/StatusHelp.jsx]: please also fix this spelling here for "relevant"

**Review by FaithMo — CHANGES_REQUESTED** [review]
I like that you moved StatusHelp to a file of its own

**Review by FaithMo — APPROVED** [review]

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
I'm getting this warning:

```
index.js:1 Warning: Failed prop type: The prop `xs` of `Grid` must be used on `item`.
    in WithStyles(ForwardRef(Grid)) (at Presentation.jsx:58)
    in Presentation (at AddCardReviewModal/index.jsx:53)
    in AddReviewModalUnconnected (created by ConnectFunction)
    in ConnectFunction (at Presentation.js:155)
    in div (at Presentation.js:153)
    in Presentation (at AppHeaderAndMenu/index.js:41)
    in AppMenuUnconnected (created by ConnectFunction)
    in ConnectFunction (at App/index.js:79)
    in Routes (at App/index.js:67)
    in ThemeProvider (at App/index.js:66)
    in AppUnconnected (created by ConnectFunction)
    in ConnectFunction (at src/index.js:15)
    in Router (created by BrowserRouter)
    in BrowserRouter (at src/index.js:14)
    in Provider (at src/index.js:13)
```

Besides that it all looks good

**Review by FaithMo — COMMENTED** [review]
I did not experience the error,

**Comment by Ngoakor12** [comment]
> I did not experience the error,

I fixed it. A new warning popped up in the browser console whenever you clicked the "add review" button on a card.

**Review by FaithMo — APPROVED** [review]

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**